### PR TITLE
updated links in serverless articles

### PR DESCRIPTION
**Pull request by etelsv**

### CODE REVIEW

File: content/11-serverless/02-serverless-comparison.mdx [hunk 1]
```diff
@@ -562,6 +573,12 @@ Google's serverless database offering is [Firestore](https://cloud.google.com/fi
 
 [PlanetScale](https://planetscale.com/) is a MySQL-compatible serverless database platform built on Vitess, a relational database implementation built for scaling in Kubernetes environments.  PlanetScale is able to provide versioned data branching, non-blocking schema changes, and other advanced features with simple to deploy tooling.
 
+<PrismaOutlinks>
+
+Learn more about how [Prisma and Planetscale work together](https://www.prisma.io/planetscale.
```
Comment (alex-emerich) [inline content/11-serverless/02-serverless-comparison.mdx:578]:
```suggestion
Learn more about how [Prisma and Planetscale work together](https://www.prisma.io/planetscale).
```

**Review by alex-emerich — CHANGES_REQUESTED** [review]

File: content/11-serverless/02-serverless-comparison.mdx [hunk 2]
```diff
@@ -306,6 +306,11 @@ Like Microsoft, Google seems to still be following Amazon's lead in the serverle
 
 Cloudflare Workers are capable of delivering high performance at a lower cost to developers due to their implementation details and some self-imposed limitations.  If your applications fit the Cloudflare Workers paradigm, it is a great choice for delivering low latency responses to your clients.
 
+<PrismaOutlinks>
```
Comment (imchairmanm) [inline content/11-serverless/02-serverless-comparison.mdx:309]: I think you might need another blank link beneath this one.  We had some rendering issues without that in the past.

File: content/11-serverless/02-serverless-comparison.mdx [hunk 3]
```diff
@@ -372,7 +377,7 @@ Another limitation inherent to the V8 runtime is that the Node.js standard libra
 
 ### Summary
 
-Cloudflare Workers is a very attractive solution for many projects due to its low latency delivery and high performance execution environment.  The platform is designed specifically to allow for quick execution of small, focused functions at the edge.
+Cloudflare Workers is a very attractive solution for many projects due to its low latency delivery and high performance execution environment.  The platform is designed specifically to allow for quick execution of small, focused functions at the [edge](/serverless/serverless-glossary#api-gateway#edge-computing).
```
Comment (imchairmanm) [inline content/11-serverless/02-serverless-comparison.mdx:380]:
```suggestion
Cloudflare Workers is a very attractive solution for many projects due to its low latency delivery and high performance execution environment.  The platform is designed specifically to allow for quick execution of small, focused functions at the [edge](/serverless/serverless-glossary#edge-computing).
```

File: content/11-serverless/02-serverless-comparison.mdx (hunk 1 above)
Comment (imchairmanm) [inline content/11-serverless/02-serverless-comparison.mdx:578]:
```suggestion
Learn more about how [Prisma and Planetscale work together](https://www.prisma.io/planetscale).
```

File: content/11-serverless/03-serverless-challenges.mdx [hunk 4]
```diff
@@ -20,7 +20,7 @@ While [serverless computing has many benefits](https://www.prisma.io/dataguide/s
 
 ## Cold start problems
 
-One of the most commonly discussed challenges when working with serverless is called the *cold start* problem.  While the goal with serverless is allow functions to immediately be executed on demand, there are some scenarios that may result in predictable delays.
+One of the most commonly discussed challenges when working with serverless is called the *[cold start](/serverless/serverless-glossary#cold-start)* problem.  While the goal with serverless is allow functions to immediately be executed on demand, there are some scenarios that may result in predictable delays.
```
Comment (imchairmanm) [inline content/11-serverless/03-serverless-challenges.mdx:23]:
```suggestion
One of the most commonly discussed challenges when working with serverless is called the [*cold start*](/serverless/serverless-glossary#cold-start) problem.  While the goal with serverless is allow functions to immediately be executed on demand, there are some scenarios that may result in predictable delays.
```

File: content/11-serverless/02-serverless-comparison.mdx [hunk 5]
```diff
@@ -306,6 +306,11 @@ Like Microsoft, Google seems to still be following Amazon's lead in the serverle
 
 Cloudflare Workers are capable of delivering high performance at a lower cost to developers due to their implementation details and some self-imposed limitations.  If your applications fit the Cloudflare Workers paradigm, it is a great choice for delivering low latency responses to your clients.
 
+<PrismaOutlinks>
+You can learn how to deploy a Cloudflare Worker that uses Prisma to save every request to a MongoDB database for inspection later in the [Prisma docs](https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-cloudflare-workers).
+
+</PrismaOutlinks
```
Comment (imchairmanm) [inline content/11-serverless/02-serverless-comparison.mdx:312]:
```suggestion
</PrismaOutlinks>
```

File: content/11-serverless/02-serverless-comparison.mdx (hunk 2 above)
Comment (imchairmanm) [inline content/11-serverless/02-serverless-comparison.mdx:309]:
```suggestion
<PrismaOutlinks>

```

**Review by imchairmanm — APPROVED** [review]

**Review by alex-emerich — APPROVED** [review]

**Pull request merged**